Project-3/Peihong_Xie_Project_3_50375781.md

```diff
@@ -0,0 +1,381 @@
+
+(d) The object property **"has skeleton"** should be functional, though it is not listed as so. According to its definition, "has skeleton" is a relation between a segment or subdivision of an organism and the maximal subdivision of material entities that provides structural support for that segment or subdivision. Given that the maximal subdivision of material entities for any organism has only one, this object property is thereby functional.
+
+(e) The object property **"overlaps"** should be both symmetric and reflexive, though it is not listed as so. First, if A overlaps B, then A and B has some common part. So B and A has also some common part, and thus B overlaps A. That is, overlapping is symmetric. Second, an entity can overlap itself because it clearly has the same parts with itself. Therefore, overlapping is also reflexive.
+
+Similarly, **"spatially coextensive with" and "simultaneous with"** is also symmetric and reflexive.
+
+**[2]** In BFO and RO identify at least one object property for each of a-e that _should not have the listed property, but which does_; argue for your case, using examples. 
+```
+  (a)  Irreflexive
+  (b)  Transitive 
+  (c)  Asymmetric
+  (d)  Functional 
+  (e)  Inverse Functional
+```
+
+**Answer:**
+
+(a) Although **"has substance added"** is listed as irreflexive, but it should not be so. That is, in some cases, a physical entity can have itself as substance added to it. For example, a portion of water can be added to itself at some time. Similarly, **"has substance removed"** should not be irreflexive, either because we can always remove the same amount of water from itself.
+
+(b) Although **"aligned with"** is listed as transitive, but it should not be so. Suppose that we have three different quadrilateral objects A, B, and C. A is aligned with B because of their alighed long sides, and B is aligned with C because of their aligned short sides, but A is not thus aligned with C because neither their long nor short sides are not aligned with each other.
+
+Another funny example: Although **"end after"** is listed as transitive, but it should not be so. A possible counterexample can be found in a spacetime circle: The event of Jack's killing his grandfather ends after the event of his shooting, and the latter ends after the event of his returning to the past. However, the event of Jack's killing his grandfather does not end after the event of his returning to the past. Similarly, **"before"** should not be transitive, either. *(paradox of time travel)*
+
+(c) Although **"has role in modeling" (or "is used to study")** is listed as asymmetric, but it should not be so. Suppose we wanna study the relation between a biological organism, say, a dog, and its biological niche. Given the mutual dependence between them in some plausible sense, it seems that the dog has role in modeling its niche, but its niche has also role in modeling it.
```

Comment:
It would be good to mention the parts about "biological, experimental, or computational artifact" for your explanation here. It's useful terminology, and you may have an issue with your current example as the dog and their niche are both not artifacts (at least under traditional understandings) since they were not intentionally created by anyone. You could keep it but using the different types of artifacts in any case would make the explanation stronger.